aspnetcore/blazor/hosting-models.md

```diff
@@ -178,7 +179,12 @@ The following Razor page renders a `Counter` component:
 ```cshtml
 <h1>My Razor Page</h1>
 
-@(await Html.RenderComponentAsync<Counter>(RenderMode.ServerPrerendered))
+<component type="typeof(Counter)" render-mode="ServerPrerendered" param-InitialValue="InitialValue" />
```

Comment:
Is it ok if I break this line (and Line 202 below)? They run over our 85 char limit to prevent horizontal scrollbars in the docs.

Reply:
Go for it